pkg/catalogv2/http/client.go

```diff
@@ -55,81 +55,13 @@ func HelmClient(secret *corev1.Secret, caBundle []byte, insecureSkipTLSVerify bo
 		Timeout:   30 * time.Second,
 	}
 	if username != "" || password != "" {
-		client.Transport = &basicRoundTripper{
-			username:               username,
-			password:               password,
-			disableSameOriginCheck: disableSameOriginCheck,
-			repoURL:                repoURL,
-			next:                   client.Transport,
-		}
+		client.Transport = roundtripper.NewDisableSameOriginRoundTripper(
```

Comment:
Not sure whether `NewInsecureDisableSameOriginRoundTripper` or something like `NewInsecureSameOriginBypassRoundTripper`.

Reply:
NewInsecureSameOriginBypassRoundTripper looks better. Done.